site/content/components/AuthorPosts.jsx

```diff
@@ -36,8 +36,11 @@ export function AuthorPosts({ id, name }) {
                   )}
                 </p>
                 <a href={`/${post.url_path}`} className="mt-2 block no-underline">
-                  <p className="text-xl font-semibold font-headings text-gray-900 underline">{post.title}</p>
-                  <PostBody code={post.body.code} frontmatterImage={post.image} />
+                  <p className="text-lg font-semibold font-headings text-primary underline">{post.title}</p>
+                  {post.description
+                    ? <p className="mt-3 text-base text-gray-500">{post.description}</p>
+                    : <PostBody code={post.body.code} frontmatterImage={post.image} className="text-gray-500" />
```

Comment:
I'm not sure why we created the `PostBody` component and the `getPageBody` function in the first place. I mean, I understand what it does, but I don't think our code should be responsible for creating a description of the blog post for the author if he/she forgot to add one. I would just simply not display it if it weren't provided. And if we do want to automatically extract the description if it's not provided, it shouldn't be a job of either `AuthorPosts` component or (the more so) its child component, but rather should be handled e.g. in the getter function. This is what I think is called a "design by contract" approach (https://en.wikipedia.org/wiki/Design_by_contract#Description).

But maybe I'm not getting sth 😄

Reply:
+1 from me.

That said: given we want to move relatively quickly: if this is a <1h fix do the fix. Otherwise, just merge as is and we will return to this later.